src/main/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiService.scala

```diff
@@ -84,6 +84,13 @@ trait AdminApiService extends HttpService with PerRequestCreator with UserInfoDi
       }
     } ~
     path("admin" / "groups" / Segment / "members") { (groupName) => //add members to group
+      put {
+        entity(as[RawlsGroupMemberList]) { memberList =>
+          requestContext => perRequest(requestContext,
+            UserService.props(userServiceConstructor, userInfo),
+            UserService.OverwriteGroupMembers(RawlsGroupRef(RawlsGroupName(groupName)), memberList))
```

Comment:
I am saying that is what our post. I could see the argument of collapsing the POST and DELETE into PATCH but we are not going to do that now. If you feel strongly I can enter techdebt JIRA.